tests_py36/test_py36.R

```diff
@@ -0,0 +1,8 @@
+context("py36")
+
+test_that("python 3.6 conda env is installed", {
+    Sys.setenv(RETICULATE_PYTHON = "/opt/conda/envs/py36/bin/", required = T)
```

Comment:
Is this going to work in the regular Docker image env? (do we set/not set this env variable in our regular Sessions as intended?)

Reply:
We have two separate python installations that we need to support inside the R image: 2.7 for tensorflow (reticulate env variable is set to this by default) and 3.6 for the competitions time series api (which this test is checking). Once you set this variable reticulate seems unwilling to let you change it inside of the R session, hence why I had to pull it into it's own test that starts a fresh R session.

Reply:
How do people switch between the two then in the kernel editor?

TensorFlow 1.15 works with python 3.6, did you look into why 2.7 is used in R?

Reply:
Didn't look into what it'd take to get tf to 3.6 in the R image. Assumed that would turn into a larger chunk of work. It's my understanding of reticulate that folks can't switch between python versions the way we are running things.

Reply:
If a user can't switch between python versions, how is this going to fix the issue with competitions time series?

Reply:
They can set a preference once via an env variable
`Sys.setenv(RETICULATE_PYTHON = '/opt/conda/envs/py36/bin/python', required = T)`
but subsequent calls to change python seem to be ignored by reticulate.

Reply:
Yeah, tried that and couldn't get to work. The package author also suggested it isn't possible in https://github.com/rstudio/reticulate/issues/45#issuecomment-293735624